exporters/otlp/otlptrace/internal/tracetransform/span.go

```diff
@@ -111,6 +111,12 @@ func span(sd tracesdk.ReadOnlySpan) *tracepb.Span {
 		s.ParentSpanId = psid[:]
 	}
 
+	flags := tracepb.SpanFlags_SPAN_FLAGS_TRACE_FLAGS_MASK | tracepb.SpanFlags_SPAN_FLAGS_CONTEXT_HAS_IS_REMOTE_MASK
+	if sd.SpanContext().IsRemote() {
```

Comment:
According to [proto definition comment](https://github.com/open-telemetry/opentelemetry-proto/blob/342e1d4c3a1fe43312823ffb53bd38327f263059/opentelemetry/proto/trace/v1/trace.proto#L120) we need to check the **parent** SpanContext here, instead of the SpanContext of the span that is being exported:

`sd.Parent().IsRemote()`

Reply:
We don't have the parent span context available (and having it is [not part of the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spancontext)).

But `IsRemote` should mean the same thing, based on the spec:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#isremote

> An API called IsRemote, that returns a boolean value, which is true if the SpanContext was propagated from a remote parent

Reply:
But the ReadOnlySpan has `.Parent()` which returns the parent SpanContext